docs/literate/src/files/differentiable_programming_literate.jl

```diff
@@ -330,7 +328,6 @@ ode = semidiscretize(semi, (0.0, 1.5));
 sol = solve(ode, BS3(), save_everystep=false);
 
 plot(sol);
```

Comment:
```
┌ Warning: #= /home/runner/work/Trixi.jl/Trixi.jl/src/solvers/dgsem/interpolation.jl:129 =#:
│ `LoopVectorization.check_args` on your inputs failed; running fallback `@inbounds @fastmath` loop instead.
│ Use `warn_check_args=false`, e.g. `@turbo warn_check_args=false ...`, to disable this warning.
└ @ Trixi ~/.julia/packages/LoopVectorization/asUOz/src/condense_loopset.jl:825
```
This warning is shown here in the documentation. Any ideas on how to fix the problem or to hide the warning?

Reply:
Not in a "nice way" without rewriting all loops in `src/solvers/dgsem/interpolation.jl`. The alternative that we might also use in other places could be to wrap some `solve` calls etc. in 
```julia
redirect_stdout(devnull) do
  # code that prints annoying stuff we don't want to see here
end
```